client/package.json

```diff
@@ -49,7 +49,7 @@
     "jest-styled-components": "^7.0.2",
     "msw": "^0.25.0",
     "prettier": "^2.0.5",
-    "react-scripts": "4.0.2",
+    "react-scripts": "4.0.3",
```

Comment:
Can you remove the `resolutions` section below along with this package update.  Then it will close https://github.com/o2Labs/office-booker/issues/136

Reply:
Removed `resolutions `